### PR TITLE
Create initial sitemode record in readwrite DB mode

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: opendebates/utils.py [hunk 1]
```diff
@@ -133,7 +134,11 @@ def sort_list(citations_only, sort, ideas):
 
 
 def get_site_mode():
-    return SiteMode.objects.get_or_create()[0]
+    try:
+        return SiteMode.objects.get()
+    except SiteMode.DoesNotExist:
+        with readwrite_db():
+            return SiteMode.objects.get_or_create()[0]
```
Comment (vkurup) [inline opendebates/utils.py]: This can just be `create()` now.

**Comment by vkurup** [comment]
1 comment then :+1:

File: opendebates/utils.py (hunk 1 above)
Comment (dpoirier) [inline opendebates/utils.py]: We could be in a race with another process or server trying to do this at the same time.

Comment (vkurup) [inline opendebates/utils.py]: ahh, ok

**Pull request merged**